docker-compose.yaml

```diff
@@ -75,6 +76,10 @@ services:
       DATABASE_PASSWORD: ${DATABASE_PASSWORD}
       DATABASE_DIALECT: ${DATABASE_DIALECT}
 
+      CACHE_NAME: ${REDIS_DATABASE}
```

Comment:
Se for o nome, poderiamos seguir com o mesmo padrão das outras variaveis, adicionando _NAME?